clients/filesystem-hadoop/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+
+plugins {
+  `maven-publish`
+  id("java")
+  id("idea")
+}
+
+val defaultHadoopVersion: Int = 3
```

Comment:
In the short term, this seems to be sufficient because the API is compatible enough. But I don't have a strong inclination whether to do it or not. Maybe we can split the modules in the beginning.

Reply:
Let's remove the hadoop version code you recently added here. I would think about how to handle it properly in another issue.